### PR TITLE
fix badges to reflect open-source status

### PR DESCRIPTION
**Pull request by vadim-ex**
@dlecocq @b4hand

### CODE REVIEW

**Comment by vadim-ex** [comment]
Thank you!

**Comment by vadim-ex** [comment]
Sorry, you'd need to merge PR. I do not have permissions.

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ Tests are run with the top-level `Makefile`:
 make test
 ```
 
-It is also possible to run then as 
+It is also possible to run then as
 
 ```bash
 python setup.py test
```
Comment (b4hand) [inline README.md]: The Contribution guidelines should be replaced with some inline text rather than referencing a document that is not external facing.

Comment (vadim-ex) [inline README.md]: I agree. I noticed it (after submitting the PR), but I could not do it myself.

**Pull request merged**